tests/test_config.py

```diff
@@ -19,7 +19,7 @@
     'default': {'url': 'http://example.com'},
     'Ask Aak': {'url': 'bogus value', 'auth': ['username', 'password']},
     'Acros-Krik': {'url': 'booger', 'version': '1.2.3.4.dev5+67'},
-    'King Adas': {'url': 'burger', 'version': 'silly version'},
+    'King Adas': {'url': 'burger', 'version': '987.654'},
```

Comment:
No, this is not the right way to fix it. You could create another test with invalid version `silly version` that verifies pytest raises InvalidVersion for incorrect version.

Reply:
Going to remove this line. Will need to add to a separate config and use that for a test asserting on the error.